manifests/mcp/02_deployment.yaml

```
@@ -0,0 +1,40 @@
+# creats a 1 pod deployment for the cluster-health-analyzer
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    app.kubernetes.io/name: cluster-health-analyzer-mcp
+  name: cluster-health-analyzer-mcp
+  namespace: openshift-cluster-observability-operator
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: cluster-health-analyzer-mcp
+  template:
+    metadata:
+      labels:
+        app.kubernetes.io/name: cluster-health-analyzer-mcp
+    spec:
+      serviceAccountName: cluster-health-analyzer-mcp-thanos-querier
+      automountServiceAccountToken: true
+      containers:
+      - name: cluster-health-mcp-server
+        image: "quay.io/openshiftanalytics/cluster-health-analyzer:latest"
```

let's pin a dedicated tag

```suggestion
        image: "quay.io/openshiftanalytics/cluster-health-analyzer:mcp-dev-preview"
```